framework/base/Module.php

```diff
@@ -239,7 +239,7 @@ public function getControllerPath()
 
     /**
      * Returns the directory that contains the view files for this module.
-     * @return string the root directory of view files. Defaults to "[[basePath]]/views".
+     * @return string the root directory of view files. Defaults to `[[basePath]]/views`.
```

Comment:
because with `"` the link defined by [[ will be parsed and transformed into a link. with this change [[ is within a code block and will not be parsed anymore.